syntax = "proto3";

package user;
import "google/protobuf/any.proto";

message UserData {
  google.protobuf.Any data = 1;
}
message InsertResponse {
  bool ok = 1;
  string message = 2;
}

service UserService {
  rpc InsertUserData (UserData) returns (InsertResponse);
}